e2e/cypress/integration/visual_regression/system_console/environment_section_spec.js

```diff
@@ -0,0 +1,122 @@
+            url: 'admin_console/environment/image_proxy',
+        },
+        {
+            section: 'Environment',
+            header: 'SMTP',
+            sidebar: 'SMTP',
+            url: 'admin_console/environment/smtp',
+        },
+        {
+            section: 'Environment',
+            header: 'Push Notification Server',
+            sidebar: 'Push Notification Server',
+            url: 'admin_console/environment/push_notification_server',
+        },
+        {
+            section: 'Environment',
+            header: 'High Availability',
+            sidebar: 'High Availability',
+            url: 'admin_console/environment/high_availability',
+        },
+        {
+            section: 'Environment',
+            header: 'Rate Limiting',
+            sidebar: 'Rate Limiting',
+            url: 'admin_console/environment/rate_limiting',
+        },
+        {
+            section: 'Environment',
+            header: 'Logging',
+            sidebar: 'Logging',
+            url: 'admin_console/environment/logging',
+        },
+        {
+            section: 'Environment',
+            header: 'Session Lengths',
+            sidebar: 'Session Lengths',
+            url: 'admin_console/environment/session_lengths',
+        },
+        {
+            section: 'Environment',
+            header: 'Performance Monitoring',
+            sidebar: 'Performance Monitoring',
+            url: 'admin_console/environment/performance_monitoring',
+        },
+        {
+            section: 'Environment',
+            header: 'Developer Settings',
+            sidebar: 'Developer',
+            url: 'admin_console/environment/developer',
+        },
+    ];
+
+    before(() => {
+        // * Check if server has license for feature
+        cy.requireLicenseForFeature('Elasticsearch');
+
+        // # Go to system admin then verify admin console URL and header
+        cy.visit('/admin_console/about/license');
+        cy.url().should('include', '/admin_console/about/license');
+        cy.get('.admin-console', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').within(() => {
+            cy.get('.admin-console__header').should('be.visible').and('have.text', 'Edition and License');
+        });
+    });
+
+    testCases.forEach((testCase) => {
+        it(`${testCase.section} - ${testCase.header}`, () => {
+            // # Click the link on the sidebar
+            cy.get('.admin-sidebar').should('be.visible').within(() => {
+                cy.findByText(testCase.sidebar).scrollIntoView().should('be.visible').click();
+            });
+
+            // * Verify that it redirects to the URL and matches with the header
+            cy.url().should('include', testCase.url);
+            cy.get('.admin-console').should('be.visible').within(() => {
+                cy.get('.admin-console__header').should('be.visible').and(testCase.headerContains ? 'contain' : 'have.text', testCase.header);
+            });
+        });
+    });
+});
```

Comment:
There is no visual snapshot in this file. Should it be outside of the `@visual_regression` group?

Reply:
Good catch! I also realized that this was unintentionally checked in since I have it on my on going dev but it's fine to remain as is. I will have follow up PRs and will add corresponding metadata and visual tests.